src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -38,7 +38,7 @@ final class ExposureNotificationSettingViewController: UITableViewController, Ac
 		navigationItem.title = AppStrings.ExposureNotificationSetting.title
```

Comment:
Same question here: Would it be possible to have the 3 lines of configuration just in viewWillAppear?

Reply:
General remarks on navigation items and navigation bar: see above.

Strangely, here it is not possible to move the lines into viewWillAppear, it has an impact on the animation, and would initially be shown with normal title.

I cannot tell for sure which fact causes this difference in behaviour, although I suspect that is because ExposureNotificationSettingViewController is inherited from UITableViewController.